docs/source/getting_started/overview.rst

```diff
@@ -85,4 +85,10 @@ We appreciate feedback on our `GitHub issues page <https://github.com/jupyterlab
 as we evolve towards a stable extension development API.
 
 JupyterLab will eventually replace the classic Jupyter Notebook.
-Throughout this transition, the same notebook document format will be supported by both the classic Notebook and JupyterLab.
+Throughout this transition, the same notebook document format will
```

Comment:
> JupyterLab will eventually replace the classic Jupyter Notebook.

Wondering if the sentence above is still valid? With the current state of the projects Notebook 7 will be the one replacing the classic notebook 6.

Reply:
But it's true we should indeed still mention JupyterLab 4 will not have the `notebook` dependency.

Reply:
Right, I think that sentence could be changed to better reflect the state of projects today. I have made some updates which I thought might fit better, but I would appreciate feedback on them!